rules-tests/Php83/Rector/BooleanAnd/JsonValidateRector/Fixture/json_validate_base.php.inc

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Rector\Tests\Php83\Rector\BooleanAnd\JsonValidateRector\Fixture;
+
+if (json_decode($json) !== null && json_last_error() === JSON_ERROR_NONE){
```

Comment:
Please add a test for yoda style comparisons aka. `null !== json...)`

Reply:
Ok